app/mailers/msr_mailer.rb

```diff
@@ -1,14 +1,20 @@
 class MsrMailer < ApplicationMailer
+	# **Have to check this**
```

Comment:
yea I'll clean it up. Also I won't merge this rn cus Zaineb said some things need to change and I will make the changed and merge them all together.